pyfstat/utils/sft.py

```diff
@@ -45,11 +54,11 @@ def get_sft_as_arrays(sftfilepattern, fMin=None, fMax=None, constraints=None):
     sft_catalog = lalpulsar.SFTdataFind(sftfilepattern, constraints)
     ifo_labels = lalpulsar.ListIFOsInCatalog(sft_catalog)
 
-    logger.info(
+    logger.debug(
         f"Loading {sft_catalog.length} SFTs from {', '.join(ifo_labels.data)}..."
     )
```

Comment:
I'd say that's a useful bit at info at info level, and consistent with the typical verbosity level of most lalpulsar tools. Strong reasons to downgrade it, to avoid excessive spamming in some particular workflow scenario?

(Next two downgrades are fine.)

Reply:
Since we are loading things into dictionaries and all that, I guess such log wouldn't be too much of a burden.